sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -417,18 +413,12 @@ class CodeGenContext {
       val code = expr.gen(this)
       val fn =
         s"""
-           |private void $fnName(InternalRow ${INPUT_ROW}) {
-           |  if (!$isLoaded) {
```

Comment:
Nvm, what I said doesn't apply for this operator.

I was thinking for Filter you wouldn't want to call fnName once for each row. It can be called less than once.

LGTM